hippod/api_cache_achievements.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+import datetime
+import time
+import sys
+
+import aiohttp
+
+import hippod.hasher
+import hippod.ex3000
+
+from hippod.error_object import *
+
+
+def check_already_listed(buffer_stat, cat):
+    listed = False
+    for i, name in enumerate(d['name'] for d in buffer_stat['children']):
```

Comment:
simple return true if found, why iterating over all elements when the result is clear?